obsei/preprocessor/base_text_cleaner.py

```diff
@@ -0,0 +1,39 @@
+from abc import abstractmethod
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, PrivateAttr
+
+
+class TextCleanerRequest(BaseModel):
+    text: str
+
+
+class TextCleanerResponse(BaseModel):
+    text: str
+    processed_text: str
+
+
+class BaseTextCleanerConfig(BaseModel):
+    TYPE: str = "Base"
+
+    class Config:
+        arbitrary_types_allowed = True
+
+
+class BaseTextCleaner(BaseModel):
+    TYPE: str = "Base"
+
+    def __init__(self, **data: Any):
+        super().__init__(**data)
+    
+    @abstractmethod
```

Comment:
Please change this to 
```
def clean_input(
        self,
        input_list: List[AnalyzerRequest],
        config: BaseTextCleanerConfig,
        **kwargs
) -> List[AnalyzerRequest]:
```

Reply:
As this will be used as follows -
Observer (Source) -> Cleaner -> Analyzer -> Informer (Sink)

Output of `Observer (Source)` and input of `Analyzer` is `AnalyzerRequest